config.yml

```diff
@@ -54,6 +54,35 @@ networks:
         transaction_timeout: 1s
         minimum_confirmations: 1
         gas_estimation_buffer: 0
+    ethereum_ganache:
+        name: "Ethereum Ganache"
+        chain_id: 1337
+        type: evm
+        private_keys:
+            - 4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
```

Comment:
Can add keys from here and just reuse as anchor https://github.com/smartcontractkit/chainlink-node-compose/blob/main/docker-compose/services.yml